docs/content/concepts/visualizers-and-overrides.md

```diff
@@ -0,0 +1,169 @@
+  <source media="(max-width: 480px)" srcset="https://static.rerun.io/viscomp-base/dbe3202b78ff44e35dab0844b3494e3a68c7e551/480w.png">
+  <source media="(max-width: 768px)" srcset="https://static.rerun.io/viscomp-base/dbe3202b78ff44e35dab0844b3494e3a68c7e551/768w.png">
+  <source media="(max-width: 1024px)" srcset="https://static.rerun.io/viscomp-base/dbe3202b78ff44e35dab0844b3494e3a68c7e551/1024w.png">
+  <source media="(max-width: 1200px)" srcset="https://static.rerun.io/viscomp-base/dbe3202b78ff44e35dab0844b3494e3a68c7e551/1200w.png">
+</picture>
+
+In the Rerun viewer, visualizations happen within _views_, which are defined by their [_blueprint_](blueprint.md).
+
+The first step for a view to display its content is to determine which entities are involved.
+This is determined by the [entity query](../reference/entity-queries.md), which is part of the view blueprint.
+The query is run against the data store to generate the list of view entities.
+
+Views rely on visualizers to display each of their entities. For example, [3D views](../reference/types/views/spatial3d_view.md) use the `Points3D` visualizer to display 3D point clouds, and [time series views](../reference/types/views/time_series_view.md) use the `SeriesLine` visualizer to display time series line plots. Which visualizers are available is highly dependent on the specific kind of view. For example, the `SeriesLine` visualizer only exist for time series views—not, e.g., 3D views.
+
+For a given view, visualizers are selected for each of its entities based on their content.
+By default, visualizers are selected for entities logged with a corresponding [archetype](../reference/types/archetypes.md).
+For example, in a 3D view, an entity logged with the [`Points3D`](../reference/types/archetypes/points3d.md) results in the `Points3D` visualizer being selected by default.
+This happens because [archetypes](../reference/types/archetypes.md) include an _indicator component_ to capture the intent of the logging code.
+This indicator component in turn triggers the default activation of the associated visualizer.
+(We will see that this process can be influenced by both the user interface and the blueprints.)
+
+Then, each selected visualizer determines the values for the components it supports. For example, the `Points3D` visualizer handles, among others, the [`Position3D`](../reference/types/components/position3d.md), [`Radius`](../reference/types/components/radius.md), and [`Color`](../reference/types/components/color.md) components. For each of these (and the others it also supports), the visualizer must determine a value. By default, it will use the value that was logged to the data store, if any. Otherwise, it will use some fallback value that
+ depends on the actual type of visualizer and view. (Again, we will see that this can be influenced by the user interface and the blueprint.)
```

Comment:
I'd drop that here since it sounds like we're influencing the fallback.
```suggestion
 depends on the actual type of visualizer and view.
```
Also this is already the longest block without picture or snippet :)